src/components/views/elements/ImageView.tsx

```diff
@@ -114,6 +114,8 @@ export default class ImageView extends React.Component<IProps, IState> {
 
     componentWillUnmount() {
         this.focusLock.current.removeEventListener('wheel', this.onWheel);
+        window.removeEventListener("resize", this.calculateZoom);
+        this.image.current.removeEventListener("load", this.calculateZoom);
```

Comment:
Nice catch - thanks!

Reply:
We may want to move this listener cleanup to a separate PR... We should definitely land this listener fix sooner, and it's unclear to me when / if the behaviour changes in the rest of this PR would land.

Reply:
Okay, I'll make a PR for that soon.